cmd/mirror.go

```diff
@@ -190,10 +191,33 @@ func delComp(repo, id, version string) error {
 	return nil
 }
 
-// the `repo publish` sub command
+// the `mirror set` sub command
+func newMirrorSetCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "set <mirror-addr>",
+		Short: "set mirror address",
+		Long:  "set mirror address, will replace the root certificate, which is risky",
```

Comment:
I think there is no risk in most scenarios, we shouldn't fetch the `root.json` every time.
Please refer the https://github.com/pingcap/tiup/issues/433